dotCMS/src/main/java/com/dotmarketing/portlets/cmsmaintenance/ajax/CMSMaintenanceAjax.java

```diff
@@ -429,128 +431,146 @@ public void createXMLFiles() throws ServletException, IOException, DotDataExcept
 			try {
 
 				/* get a list of all our tables */
-				Map map = HibernateUtil.getSession().getSessionFactory().getAllClassMetadata();
+				final Map map = new HashMap();
+
+				//Including Identifier.class because it is not mapped with Hibernate anymore
+				map.put(Identifier.class, null);
+
+				map.putAll(HibernateUtil.getSession().getSessionFactory().getAllClassMetadata());
+
 				Iterator it = map.entrySet().iterator();
 				while (it.hasNext()) {
 					Map.Entry pairs = (Map.Entry) it.next();
 					Class x = (Class) pairs.getKey();
-					if (!x.equals(Inode.class) && !x.equals(Clickstream.class) && !x.equals(ClickstreamRequest.class)
-					        && !x.equals(Plugin.class) && !x.equals(PluginProperty.class))
+					if (!x.equals(Inode.class) && !x.equals(Clickstream.class) && !x
+							.equals(ClickstreamRequest.class)
+							&& !x.equals(Plugin.class) && !x.equals(PluginProperty.class)) {
 						_tablesToDump.add(x);
+					}
 
 				}
 				XStream _xstream = null;
 				HibernateUtil _dh = null;
+				DotConnect _dc = null;

```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)